starknet_py/contract.py

```diff
@@ -244,7 +250,7 @@ async def deploy_v3(
         :param unique: Determines if the contract should be salted with the account address.
         :param constructor_args: a ``list`` or ``dict`` of arguments for the constructor.
         :param nonce: Nonce of the transaction with call to deployer.
-        :param l1_resource_bounds: Max amount and max price per unit of L1 gas (in Fri) used when executing
+        :param resource_bounds: Max amount and max price per unit of L1 and L2 gas (in Fri) used when executing
```

Comment:
This description needs to be updated in all places (perhaps in the docs as well)